.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!--
+Please use this issue tracker only for reporting bugs or feature requests.
+
+If you need support, please use:
+ 
+- our community forum
+  http://discourse.algolia.com
+  
+- StackOverflow with the `algolia` tag
+  https://stackoverflow.com/questions/tagged/algolia
+
+-->
+
+- Algolia Client Version: #.#.#
```

Comment:
Would rather ask for Django integration version 😛

Reply:
Also ask for Django version

Reply:
I didn't personalize them yet, I'll work on it :) Thanks for the feedback